db/model.js

```diff
@@ -0,0 +1 @@
+module.exports.endpoints = require("../endpoints.json")
```

Comment:
well done for directly requiring in the endpoints.json file to access the data rather than reading it - much quicker!